Code/Framework/AzFramework/AzFramework/IO/LocalFileIO.cpp

```diff
@@ -829,8 +829,21 @@ namespace AZ
                     {
                         return false;
                     }
-                    replacedAliasPath.Native() = AZ::IO::FixedMaxPathString(AZStd::string_view{ aliasValue });
-                    replacedAliasPath.Native() += postAliasView;
+
+                    // Check if the path is input path does not overlap the output path
```

Comment:
nit: Should be "// Check if the input path does not overlap the output path

Reply:
Will fix